ambry-clustermap/src/test/java/com.github.ambry.clustermap/TestUtils.java

```diff
@@ -200,7 +202,8 @@ static JSONArray getJsonArrayDataNodesRackAware(int dataNodeCount, String hostna
   private static void updateJsonArrayDataNodesRackAware(JSONArray dataNodeJsonArray, int dataNodeCount, String hostname,
       int basePort, int sslPort, int numRacks, HardwareState hardwareState, JSONArray disks) throws JSONException {
     for (int i = dataNodeJsonArray.length(); i < dataNodeCount; ++i) {
-      dataNodeJsonArray.put(getJsonDataNode(hostname, basePort + i, sslPort + i, i % numRacks, hardwareState, disks));
+      dataNodeJsonArray.put(
+          getJsonDataNode(hostname, basePort + i, sslPort + i, i % numRacks, 64, hardwareState, disks));
```

Comment:
why not use a constant value at the beginning of this file? Then we can apply it in different places.

Reply:
Done.